index.html

```diff
@@ -1343,115 +1343,92 @@
-unacceptable under another set, so when an API designer intends to explain that their new
+unacceptable under another set. When an API designer intends to explain that their new
 API is acceptable because an existing acceptable API already exposes the same information,
 they must be careful to ensure that their new API is only available under a set of guards
-that's at least as strict. Without those guards, they need to make the argument from
+that are at least as strict. Without those guards, they need to make the argument from
 scratch, without relying on the existing API.
 
-### Handling information that's being removed {#unacceptable-information}
+If existing APIs provide access to some information, but there is a plan to
+change those APIs to prevent that access, new APIs must not be added that
+provide that same information, unless it is possible to put additional
+[=appropriate=] [=access guards=] in place.
 
-<div class="practice">
-  <span class="practicelab">If existing APIs provide access to some information, but
-  we have a plan to change those APIs to prevent that access, new APIs must not be added
-  that provide that same information without extra [=access guards=] that make the access to the
-  information acceptable.</span>
-</div>
+For example, browsers are gradually removing the ability to join identities
+between different [=partitions=]. It is important that new APIs do not add
+features which re-enable [=cross-context recognition=].
 
-### Handling information we can't completely block {#sketchy-information}
+Some functionality of the web has historically been provided using features
+that can be used to undermine people's privacy (e.g. third-party cookies). It
+is not yet possible to remove access to all of the information that it would be
+better not to expose.
 
-<div class="practice">
-  <span class="practicelab">New APIs that provide access to undesirable information should
-  not make that information easier to access, unless they add [=access guards=] that make the
-  information acceptable.</span>
-</div>
+<aside class="example">
+  Some users are disappointed that the page they're visiting can discover which link
+  they clicked to leave that page. We can't block that information
+  because the page can use <a data-cite="RFC9110#status.3xx">HTTP redirects</a> to
+  learn it, and redirection is a core feature of the web.
+
+  Some users are disappointed that a page with permission to run JavaScript can record
+  their pattern of interaction with that page. However, the page does this by using
+  the same events it would use to make the page interactive, so we can't block this
+  information access either.
+</aside>
+
+New APIs that unavoidably provide access to this kind of information should
+not make that information easier to access, and nor should these APIs be
+easier to use by developers, compared to existing comparable web platform features.
+
+These APIs should also:
```

Comment:
This is just me being pedantic, so feel free to ignore if you don't want to add the words:

```suggestion
Specifications describing these APIs should also:
```

Reply:
Fewer words always better!

(Held lightly; if you think it adds clarity, more words is better. I'll leave to you.)

Reply:
+1 to the clarity that these are additions to specifications, not APIs.